compiler/src/dotty/tools/dotc/config/Properties.scala

```diff
@@ -135,4 +135,10 @@ trait PropertiesTrait {
     }
     okVersions exists (javaVersion startsWith _)
   }
+
+  // provide a main method so version info can be obtained by running this
+  def main(args: Array[String]): Unit = {
```

Comment:
Yes, something got screwed during rebase. Commit messages also got off-by-one, and the implementation of java8 check simply disappeared.